module/zfs/vdev_indirect.c

```diff
@@ -1855,6 +1859,11 @@ vdev_ops_t vdev_indirect_ops = {
 	.vdev_op_rele = NULL,
 	.vdev_op_remap = vdev_indirect_remap,
 	.vdev_op_xlate = NULL,
+	.vdev_op_rebuild_asize = NULL,
+	.vdev_op_metaslab_init = NULL,
+	.vdev_op_config_generate = NULL,
+	.vdev_op_nparity = NULL,
+	.vdev_op_ndisks = NULL,
```

Comment:
They are definitely zero-filled by default.  My personal feeling is since there aren't that many it helps readability by clearly showing what all the ops are and which ones have been implemented.